dotnet/src/webdriver/WebDriver.csproj.prebuild.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+echo "Building Selenium Manager binaries"
+bazel build //dotnet/src/webdriver:manager-linux
+bazel build //dotnet/src/webdriver:manager-windows
+bazel build //dotnet/src/webdriver:manager-macos
+
```

Comment:
Should these get put in conditionals like the ones below? If they are already built, bazel isn't going to rebuild them?

Reply:
7 seconds for invoking all dependent bazel targets. Pretty good for now, at least we can even improve it (when it will become annoying) via caching input files for msbuild targets.